src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println(this.getClass().getSimpleName() + " started work");
```

Comment:
The message should indicate what type of `Machine` is working right now. Consider including the specific action that the `Excavator` is performing to make the message more informative.